### PR TITLE
Tq42 1604 update notebooks public sdk

### PR DESCRIPTION
**Pull request by lusnig**
changed the following notebook:
- quenc_example.ipynb
- ts_generic_example.ipynb

All the other are up-to-date

### CODE REVIEW

**Comment by lusnig** [comment]
cleaned the output cells

File: notebooks/quenc_example.ipynb [hunk 1]
```diff
@@ -76,26 +75,24 @@
...
-    "            'saved_circuit': True, \n",
     "            'optimizer': 'ADAM'\n",
     "        },\n",
     "        'inputs': {}\n",
     "    }\n",
     "    run = ExperimentRun.create(\n",
     "        client=client,\n",
     "        algorithm=AlgorithmProto.TETRA_QUENC, \n",
-    "        experiment_id=exp_sample.id,\n",
+    "        experiment_id=\"e8ca73ae-d9a2-41a2-aeb8-7a734916fd72\",\n",
```
Comment (sbtq) [inline notebooks/quenc_example.ipynb:95]: this should still be `exp_sample.id`

File: notebooks/ts_generic_example.ipynb [hunk 2]
```diff
@@ -213,18 +214,22 @@
...
     "        ),\n",
+    "        data_processing_parameters=TrainDataProcessingParametersProto(\n",
+    "            input_columns=[0, 1, 2, 3],\n",
+    "            output_columns=[4]\n",
+    "        )\n",
     "    ),\n",
     "    inputs=MLTrainInputsProto(\n",
     "        # Provide the specific dataset storage ID of the data you uploaded to TQ42.\n",
-    "        data=DatasetStorageInfoProto(storage_id=\"ENTER_DATASET_STORAGE_ID_HERE\")\n",
+    "        data=DatasetStorageInfoProto(storage_id=\"1023200d-6d36-45ea-b66d-afbcc192eabc\")\n",
```
Comment (sbtq) [inline notebooks/ts_generic_example.ipynb:224]: Please also revert this line so people know they need to fill something in here.

**Review by sbtq — CHANGES_REQUESTED** [review]

File: notebooks/ts_generic_example.ipynb (hunk 2 above)
Comment (lusnig) [reply notebooks/ts_generic_example.ipynb]: did that also in all the other notebooks

File: notebooks/quenc_example.ipynb (hunk 1 above)
Comment (lusnig) [reply notebooks/quenc_example.ipynb]: done

**Review by sbtq — APPROVED** [review]

**Pull request merged**